Classes/Wegmeister/TagManager/Domain/Repository/TagRepository.php

```diff
@@ -42,7 +42,7 @@ public function findByGroups(array $groups = [], $caseSensitive = true, $cacheRe
         $constraints = [];
         // $constraints[] = $query->in('groupname', $groups);
         foreach ($groups as $group) {
-            if ($group instanceof \Wegmeister\TagManager\Domain\Model\Group) {
+            if ($group instanceof Wegmeister\TagManager\Domain\Model\Group) {
```

Comment:
Removing the \ in the beginning of the class changes the way the class is loaded. Now the class is loaded from within the current Namespace (Wegmeister\TagManager\Domain\Repository\Wegmeister\TagManager\Domain\Model\Group) which leads to a non-existing class.